src/test/ui/traits/mutual-recursion-issue-75860.stderr

```diff
@@ -5,7 +5,135 @@ LL |     iso(left, right)
    |     ^^^
    |
    = help: consider increasing the recursion limit by adding a `#![recursion_limit = "256"]` attribute to your crate (`mutual_recursion_issue_75860`)
-note: required by a bound in `Option`
+   = note: required because it appears within the type `Option<Option<_>>`
```

Comment:
This will need to be fixed before merging. Pretty sure it is part of the other PR.